kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -365,21 +352,21 @@ define([
                                 ),
                             ]
                         ),
+                        buttons,
                     ]
                 );
             return {
-                events: events,
-                content: content,
+                events,
+                content,
             };
         }
 
         function getOutdatedWarning(_cell) {
             if (utils.getCellMeta(_cell, 'kbase.appCell.outdated', false)) {
-                return a(
+                return span(
                     {
                         tabindex: '0',
-                        type: 'button',
-                        class: `${buttonClasses} ${cssBaseClass}__button ${cssBaseClass}__icon--outdated`,
+                        class: `${cssBaseClass}__icon--outdated`,
```

Comment:
remove these extra css classes and inline style -- all css is now applied to the `${cssBaseClass}__icon--outdated` selector
Also turned it into a `span` since there was no need for it to be a link element